examples/simple_structures/array_of_objects.explore.lkml

```diff
@@ -0,0 +1,15 @@
+include: "array_of_objects.view.lkml"
+   
+explore: json_table {
+  view_name: json_table
+  from: json_table
+  label: "json_table explore"
+  description: "json_table explore"
+
+  join: json_table_json {
+     from: json_table_json
+     sql:,LATERAL FLATTEN(OUTER => TRUE, INPUT => json_table."JSON") json_table_json;;
```

Comment:
Interesting approach.

* Does this also work when you just select a field from `json_table_json` in Looker, or could that break the SQL?
* Can you connect with our Looker team internally to figure out whether that's the best approach to ensure we're choiosing a good default for people to follow?

Reply:
Verifying...

Reply:
1. SQL generated is valid
2. We are generating valid Looker model with explosion logic
3. We are missing a view for the table (this view in some cases will not contain any dimensions). After adding this view, Looker is able to query the table.

I will do code corrections to fix (3)

Reply:
Fix implemented.